src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { cardsData } from '../../api/cardsData';
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import Card from './Card';
+
+describe('when rendered', () => {
+  const productStab = cardsData.products[0];
+  const { title, thumbnail, price, description, rating, brand, id } = productStab;
+  it('Should have product title', () => {
+    render(
+      <Card
+        id={id}
+        title={title}
+        brand={brand}
+        description={description}
+        price={price}
+        rating={rating}
+        thumbnail={thumbnail}
+      />
+    );
+    expect(screen.getByText(new RegExp(title, 'i'))).toBeTruthy();
```

Comment:
Why do you use regexes for text matching? `screen.getByText(title)` would be more simpler and you can control full-string or substring match with [`exact` option](https://testing-library.com/docs/queries/about/#textmatch-examples)

Reply:
fix